CMakeLists.txt

```
@@ -18,8 +18,7 @@ set(CMAKE_USER_MAKE_RULES_OVERRIDE_CXX ${CMAKE_CURRENT_SOURCE_DIR}/cmake/cxx_fla
 
 project(polserver CXX C)
```

Use more modern CMake features
```suggestion
project(polserver 
	LANGUAGES CXX C
	VERSION 100.0.0
	DESCRIPTION "Penultima Online"
	HOMEPAGE_URL "polserver.com")
```